src/openms/include/OpenMS/FILTERING/DATAREDUCTION/FeatureFindingMetabo.h

```diff
@@ -227,8 +259,12 @@ namespace OpenMS
      *
      * Reference: Kenar et al., doi: 10.1074/mcp.M113.031278
      *
+     * an alternative scoring was added which considers all possible m/z
+     * differences of all isotopes of a set of elements. The mass trace
+     * is positively scored if it could be an isotope based on the given elements.
+     *
     */
-    double scoreMZ_(const MassTrace &, const MassTrace &, Size isotopic_position, Size charge) const;
+    double scoreMZ_(const MassTrace &, const MassTrace &, Size isotopic_position, Size charge, std::vector<const Element*> elements) const;
```

Comment:
maybe use separate functions for the different scoring approaches?

Reply:
I moved the parts which are different for the scorings into separate methods.